src/app/app.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+angular.module('esn.calendar.event-consultation', [
+  'AngularJstz',
+  'ngRoute',
+  'ngSanitize',
+  'esn.avatar',
+  'esn.core',
+  'esn.calendar',
+  'esn.resource.libs',
+  'esn.calMoment',
+  'esn.constants',
+  'esn.core',
+  'esn.cache',
+  'esn.header',
+  'esn.http',
+  'esn.i18n',
+  'esn.ical',
+  'esn.lodash-wrapper',
+  'esn.notification',
+  'esn.object-type',
+  'esn.user',
+  'esn.url',  
```

Comment:
THere is no requirement from `esn-frontend-common-libs` for those `esn` modules

Reply:
These modules are required in calendar module already and this calendar public module requires calendar module. So we don't need to require it here